src/test/java/uk/ac/ebi/eva/utils/EvaJobParameterBuilder.java

```diff
@@ -17,45 +17,51 @@
 
 import org.springframework.batch.core.JobParameter;
 import org.springframework.batch.core.JobParametersBuilder;
+
 import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
 
 import java.sql.Timestamp;
 import java.util.Date;
 
-public class EvaJobParameterBuilder extends JobParametersBuilder{
+public class EvaJobParameterBuilder extends JobParametersBuilder {
 
-    public EvaJobParameterBuilder inputStudyId(String inputStudyId){
-        addParameter(JobParametersNames.INPUT_STUDY_ID, new JobParameter(inputStudyId));
+    public EvaJobParameterBuilder inputStudyId(String inputStudyId) {
+        super.addParameter(JobParametersNames.INPUT_STUDY_ID, new JobParameter(inputStudyId));
```

Comment:
because I added a generic method at the end called addParameter as well that takes a string instead of the parent's one that takes a JobParameter.